Controller/Main.as

```diff
@@ -122,6 +122,11 @@ void RefreshLeaderboard(){
             return;
         }
 
+        array<LeaderboardEntry@> clubEntries = ClubLeaderboardAPI::GetClubLeaderboardMembers(req.mapId);
```

Comment:
Thanks for the changes! 
Regarding this, I should have thought about it before, sorry; Do you think it would be valuable to also offer this for the non-api branch? If you think so and if you wish to add it, remember that you will only be able to request the positions of users who are better than you, since the endpoint used only return accurate positions for times below yours (you can check how I handle medals for that)